plugin/src/main/groovy/com/btkelly/gnag/reporters/AndroidLintViolationDetector.groovy

```diff
@@ -72,7 +74,10 @@ class AndroidLintViolationDetector extends BaseViolationDetector {
                 final String nullableCommentInHtml = notNullCommentInHtml.isEmpty() ? null : notNullCommentInHtml
 
                 final String lineNumberString = sanitizeToNonNull((String) violation.location.@line.text())
-                final Integer lineNumber = computeLineNumberFromString(lineNumberString, violationType)
+                final Integer lineNumber = LineNumberParser.parseLineNumberString(
```

Comment:
Refactored to allow Checkstyle parsing code (now shared by Checkstyle and ktlint checkers) to call these methods from outside the `reporters` package.